lib/cli.rb

```diff
@@ -349,13 +349,14 @@ def set_priorities
 EOT
   option 'board-id', desc: 'Id of the board', required: true
   option 'target-board-id', desc: 'Id of the target board', required: true
+  option 'labelled', desc: 'Set true to label cards as - in the last sprint', required: false, type: :boolean, default: false
```

Comment:
I would suggest to use another name for the option. `labelled` sounds like the label is already there. I would rather express it as an action. It also would be good to make it clear what label is added. Maybe `--set-last-sprint-label`?